### ISSUE TITLE
Feature Request: label smoothing CE loss for semantic segmentation 

### ISSUE THREAD

**Issue by seyeeet** [pytorch/pytorch#67863]
## 🚀 Feature
Recently pytorch 1.10 added label smoothing for CE (Thanks!). It would be great if we can also have label smoothing for CE for semantic segmentation. Also considering the fact that Segmentation is very similar to classification, it should be very straight forward.

cc @albanD @mruberry @jbschlosser @walterddr

**Comment by jbschlosser** [comment]
Hey @seyeeet, thanks for the request! I believe this is already supported. Consider this example:

```python
import torch

IM_HEIGHT = 200
IM_WIDTH = 200
N = 10  # batch size
C = 5  # num classes
input = torch.randn(N, C, IM_HEIGHT, IM_WIDTH)  # prediction logits per pixel
target = torch.randint(0, C, size=(N, IM_HEIGHT, IM_WIDTH))  # ground truth labels per pixel
criterion = torch.nn.CrossEntropyLoss(label_smoothing=0.1)
loss = criterion(input, target)
print(loss)
```

Or is this not what you're looking for?

**Comment by seyeeet** [comment]
yes, I  also realized that it is already in.
thanks
